api/v1/ytsaurus_types.go

```diff
@@ -571,6 +571,13 @@ type CommonSpec struct {
 	//+kubebuilder:default:=false
 	//+optional
 	UseIPv4 bool `json:"useIpv4"`
+	//+kubebuilder:default:=true
+	//+optional
+	KeepSocket bool `json:"keepSocket"`
+	//+kubebuilder:default:=false
+	//+optional
+	ForceTcp bool `json:"forceTcp"`
+
```

Comment:
force_tcp - false
keep_socket - true

cause of https://github.com/ytsaurus/ytsaurus/blob/b41294f14e346c455b1d69d268b029a5d07f71f5/yt/yt/core/dns/config.cpp#L21-L24